src/FileMutex.php

```diff
@@ -20,39 +19,16 @@ public function __construct(private readonly string $fileName)
 
     public function acquire(): Lock
     {
-        // Try to create the lock file. If the file already exists, someone else
-        // has the lock, so set an asynchronous timer and try again.
+        $f = \fopen($this->fileName, 'c');
```

Comment:
Why not `touch` here?

Reply:
This is a one-time cost, not really worth adding a touch here imo